src/fontra/views/editor/panel-transformation.js

```diff
@@ -854,71 +825,52 @@ const alignBottom = {
   },
 };
 
-const distributeHorizontally = {
-  undoLabel: "distribute horizontally",
-  distributeDirection: "horizontally",
-  computeDeltasFromBoundingBoxes: (boundingBoxes, distributeValue) => {
-    let effectiveWidth = 0;
+class DistributeObjectsDescriptor {
+  constructor(direction, directionVar) {
+    this.undoLabel = `distribute ${direction}`;
+    this.minProperty = `${directionVar}Min`;
+    this.maxProperty = `${directionVar}Max`;
+    this.deltaProperty = directionVar;
+  }
+
+  computeDeltasFromBoundingBoxes(boundingBoxes, distributeValue) {
+    let effectiveDimentions = 0;
```

Comment:
typo: `effectiveDimensions`. 

But maybe `effectiveSize` or `effectiveExtent` would be better?

Reply:
Thanks. done.